script.sh

```diff
@@ -17,16 +17,21 @@ end_section "conda.check"
 
 $SPACER
 
+echo "Install pexpect"
```

Comment:
It is installed already in the `conda-env` by `conda-build-prepare` at this point. That environment should be activated by the `common.sh`. I suggest trying without that and using python without it's full path running `.ci-output.py`.

Reply:
I applied running python without it's full path and it works. Removed additional pexpect installation